ansible/roles/apiserver/templates/naisdevice-health-checker.service.j2

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=naisdevice-health-checker
+
+[Service]
+Restart=always
+StandardOutput=append:/var/log/naisdevice/naisdevice-health-checker.json
+StandardError=append:/var/log/naisdevice/naisdevice-health-checker.json
+ExecStart=/bin/bash -c 'KOLIDE_API_TOKEN=$(gcloud secrets versions access latest --secret kolide-api-token) \
+        APISERVER_USERNAME=device-health-checker \
+        APISERVER_PASSWORD=$(gcloud secrets versions access latest --secret nais-device_device-health-checker_api-server-password) \
+        /opt/nais-device/bin/naisdevice-health-checker.phar'
```

Comment:
Put the token and password from $() in double quotes in case the password contains special chars

Reply:
Fixed in 9c6e662